pkgs/development/python-modules/cryptography/default.nix

```diff
@@ -24,9 +25,6 @@
 , setuptoolsRustBuildHook
 }:
 
-let
-  cryptography-vectors = callPackage ./vectors.nix { };
-in
```

Comment:
Also please revert this. I do not want people to use this.

Reply:
Done.